config.yaml

```diff
@@ -0,0 +1,19 @@
+AMQP_URI: 'amqp://guest:guest@localhost'
+WEB_SERVER_ADDRESS: '0.0.0.0:8000'
+rpc_exchange: 'nameko-rpc'
+max_workers: 10
```

Comment:
No, I don't think it was. I changed this because the config file now has a different purpose: be a default config file rather than an example w/ more workers (hence the new name).